### PR TITLE
COMRS-183 -Added metadata sharing package for 'FICHE DU PATIENT AVEC VIH - ENFANT' form

### PR DESCRIPTION
**Pull request by rubailly**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,76 @@
+*.class
```
Comment (tmvumbi) [inline .gitignore:1]: @rubailly the .gitignore below is used for java projects. The metadata repo will only contain config files (that we add manually); the files ignored here are artifacts created during the java build process (which won't happen for this repo). I think we can safely remove the .gitignore for the metadata repo.

**Review by tmvumbi — CHANGES_REQUESTED** [review]
@rubailly the import of the zip is failing, below are some insights from the logs: 
```
ERROR - InitializerServiceImpl.importMetadataSharingPackages(160) |2019-03-14 13:27:50,632| The MDS package could not be imported: /opt/openmrs/configuration/metadatasharing/ConceptsForFormChildPatientWithHiv-1.zip
...
Caused by: org.openmrs.api.db.DAOException: Error saving org.openmrs.Concept [a2065636-5326-40f5-aed6-0cc2cca81ccc]
...
Caused by: org.openmrs.api.APIException: Cannot save Concept #2146. Failed with the following exception:
...
Caused by: org.openmrs.api.DuplicateConceptNameException: 'Oui' is a duplicate name in locale 'fr'
```

**Review by tmvumbi — CHANGES_REQUESTED** [review]
@rubailly the import of the zip is failing, below are some insights from the logs: 
```
ERROR - InitializerServiceImpl.importMetadataSharingPackages(160) |2019-03-14 13:27:50,632| The MDS package could not be imported: /opt/openmrs/configuration/metadatasharing/ConceptsForFormChildPatientWithHiv-1.zip
...
Caused by: org.openmrs.api.db.DAOException: Error saving org.openmrs.Concept [a2065636-5326-40f5-aed6-0cc2cca81ccc]
...
Caused by: org.openmrs.api.APIException: Cannot save Concept #2146. Failed with the following exception:
...
Caused by: org.openmrs.api.DuplicateConceptNameException: 'Oui' is a duplicate name in locale 'fr'
```

File: .gitignore (hunk 1 above)
Comment (rubailly) [reply .gitignore:1]: @tmvumbi -how do you clone and make changes tho the files/folders of this repo? Me, I cloned it into Eclipse like I do with other projects and made my changes. I assume others will also try the same, like cloning the project with IntelliJ and adding their changes. 

You end-up having files like .**cproject**  which you don't want to push. We can maybe cleanup the .gitingore file and remove all the unnecessary lines, but have at least IDE related lines remain.
Unless we assume no one will ever use an IDE to work on this repo 😀

**Pull request closed without merging**